Enterprise-CMCS/eAPD
# Dmirano/4265 subform button accessibility

Resolves #4265 

### Description
Resolves a small accessibility issue for subforums when hiding the add button when a subforum is already open

### Significant changes or possible side effects
None, just added aria-labels 

### Automated test cases written
None

### Steps to manually verify this change
This was only found by the WAVE extension, so you would have to get that if you'd like to verify this change. There is a chrome extension here: https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US
(unsure about alternatives)

1. On staging, go to a page with a subforum and open it. The add button should disappear
2. Run WAVE on the page and verify it finds the "empty button" accessibility errors
3. Repeat with the preview build of this branch and verify WAVE finds no errors 

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
